src/app/public/modules/description-list/types/description-list-mode.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Specifies how to display the term-description pairs within a description list.
+ */
+export enum SkyDescriptionListMode {
+
+  /**
+   * Displays terms and descriptions vertically with the term above the description.
+   * This mode provides a responsive layout and uses the description list's `orientation`
+   * property to determine how to group term-description pairs.
+   */
+  standard = 'standard',
```

Comment:
@Blackbaud-MattGregg @Blackbaud-AdamFunderburk @blackbaud-johnly thoughts on this?

Reply:
Sorry for the slow response on this. Alex and I talked a bit offline about the terminology to use for these modes. We both agree that `stacked` is a bit ambiguous, and I had reservations about `Grid`. Here's what we came up with for the three modes:
1. `horizontal` for displaying term-descriptions horizontally side-by-side
2. `vertical` for displaying term-descriptions vertically with one on top of the other
3. `longDescription` for the longer description option that displays terms to the left of the descriptions instead of above them.

(We considered `horizontalList` and `verticalList` to make it clear that the horizontal/vertical description applies to the LIST and not LIST ITEMS, but ultimately, this doesn't seem necessary. The property is for the list, not for pairs, so `List` at the end of the property names should be redundant. We can make things clear in the docs, but feel free to weigh in if you disagree.)

Reply:
This has been done. There are now 3 `mode` settings and `orientation` has been removed.